test/python/visualization/test_circuit_matplotlib_drawer.py

```diff
@@ -35,6 +36,7 @@ def _expected_empty(self):
 
     @unittest.skipIf(not visualization.HAS_MATPLOTLIB,
                      'matplotlib not available.')
+    @unittest.skipIf(os.name == 'nt', 'tempfile fails on appveyor')
```

Comment:
I remember a conversation about temporary files in Windows... Cannot be solved by using `TemporaryFile`? 

Is this related with the depency issue?  Or a story for another PR?

Reply:
I just copied this skip from another test, there might be a way in appveyor to get tempfiles to work it's a permissions issue in the way they configure the windows environments. But I think we should fix that in another pr, just to get things green again quickly

Reply:
makes sense. I submitted https://github.com/Qiskit/qiskit-terra/issues/2335 to track it.